client/api/src/in_mem.rs

```diff
@@ -155,7 +155,7 @@ impl<Block: BlockT> Blockchain<Block> {
 	pub fn insert(
 		&self,
 		hash: Block::Hash,
-		header: <Block as BlockT>::Header,
+		header: <Block as sp_runtime::traits::HeaderProvider>::Header,
```

Comment:
There's a mix of fully qualified and non fully qualified paths to `HeaderProvider` in this file.

Reply:
Have fixed the cases where there was a mix.